[2022-07-03 10:03:37,869] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-07-03T10:03:25.858837+00:00 [queued]>
[2022-07-03 10:03:37,981] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-07-03T10:03:25.858837+00:00 [queued]>
[2022-07-03 10:03:38,003] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-03 10:03:38,006] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-03 10:03:38,012] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-03 10:03:38,355] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2022-07-03 10:03:25.858837+00:00
[2022-07-03 10:03:38,677] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'manual__2022-07-03T10:03:25.858837+00:00', '--job-id', '3', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpv1sspmdu', '--error-file', '/tmp/tmprk1vr8au']
[2022-07-03 10:03:38,700] {standard_task_runner.py:80} INFO - Job 3: Subtask also_run_this
[2022-07-03 10:03:38,655] {standard_task_runner.py:52} INFO - Started process 1582 to run task
[2022-07-03 10:03:40,954] {task_command.py:370} INFO - Running <TaskInstance: example_bash_operator.also_run_this manual__2022-07-03T10:03:25.858837+00:00 [running]> on host 5f7c6419fd02
[2022-07-03 10:04:27,200] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-07-03T10:03:25.858837+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-03T10:03:25.858837+00:00
[2022-07-03 10:04:27,427] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-03 10:04:27,523] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=manual__2022-07-03T10:03:25.858837+00:00 | dag_run=<DagRun example_bash_operator @ 2022-07-03 10:03:25.858837+00:00: manual__2022-07-03T10:03:25.858837+00:00, externally triggered: True>"']
[2022-07-03 10:04:28,388] {subprocess.py:85} INFO - Output:
[2022-07-03 10:04:28,669] {subprocess.py:92} INFO - run_id=manual__2022-07-03T10:03:25.858837+00:00 | dag_run=<DagRun example_bash_operator @ 2022-07-03 10:03:25.858837+00:00: manual__2022-07-03T10:03:25.858837+00:00, externally triggered: True>
[2022-07-03 10:04:28,701] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-03 10:04:30,886] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220703T100325, start_date=20220703T100337, end_date=20220703T100430
[2022-07-03 10:04:34,549] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-07-03 10:04:35,018] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 1582. PIDs of all processes in the group: [1582]
[2022-07-03 10:04:35,050] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 1582
[2022-07-03 10:04:36,281] {process_utils.py:75} INFO - Process psutil.Process(pid=1582, status='terminated', exitcode=0, started='10:03:38') (1582) terminated with exit code 0
